addons/project/models/project.py

```diff
@@ -190,18 +190,22 @@ def _get_default_favorite_user_ids(self):
         help="Internal email associated with this project. Incoming emails are automatically synchronized "
              "with Tasks (or optionally Issues if the Issue Tracker module is installed).")
     privacy_visibility = fields.Selection([
-            ('followers', 'Invited employees'),
-            ('employees', 'All employees'),
-            ('portal', 'Portal users and all employees'),
+            ('followers', 'Invited internal users'),
+            ('employees', 'All internal users'),
+            ('portal', 'Invited portal users and all internal users'),
         ],
         string='Visibility', required=True,
         default='portal',
-        help="Defines the visibility of the tasks of the project:\n"
-                "- Invited employees: employees may only see the followed project and tasks.\n"
-                "- All employees: employees may see all project and tasks.\n"
-                "- Portal users and all employees: employees may see everything."
-                "   Portal users may see project and tasks followed by.\n"
-                "   them or by someone of their company.")
+        help="People to whom this project and its tasks will be visible.\n\n"
+            "- Invited internal users: when following a project, internal users will get access to all of its tasks without distinction. "
+            "Otherwise, they will only get access to the specific tasks they are following.\n "
+            "A user with the project > administrator access right level can still access this project and its tasks, even if they are not explicitly part of the followers.\n\n"
+            "- All internal users: all internal users can access the project and all of its tasks without distinction.\n\n"
+            "- Invited portal users and all internal users: all internal users can access the project and all of its tasks without distinction.\n"
+            "Portal users can only access the projects and the tasks they are following. "
+            "This access can be modified on each task individually by adding or removing the portal user as follower.\n\n"
+            "In any case, an internal user with no project access rights can still access a task, "
+            "provided that they are given the corresponding URL (and that they are part of the followers if the project is private).")
```

Comment:
i don't understand this line, can you help me explain? @MissingNoShiny

Reply:
Hello, it means that if you give the url of the task to an internal user, they can view it regardless of their project access rights. If the project has the visibility "Invited internal users" however, they still need to be part of the followers of the task for this to apply.

Reply:
> Hello, it means that if you give the url of the task to an internal user, they can view it regardless of their project access rights. If the project has the visibility "Invited internal users" however, they still need to be part of the followers of the task for this to apply.

oke, i understand,thanks !

Reply:
You're welcome :slightly_smiling_face: